### PR TITLE
Sync the rpm manifest with the deb ones

### PR DESCRIPTION
**Pull request by luxas**
@dgoodwin @mikedanese PTAL

### CODE REVIEW

File: rpm/kubelet.spec [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 Name: kubelet
 Version: %{KUBE_VERSION}
-Release: %{RPM_RELEASE}
+Release: %{RPM_RELEASE}.1
```
Comment (dgoodwin) [inline rpm/kubelet.spec]: I would just bump the RPM_RELEASE counter here to signal a packaging change not an actual software change. Same for the .0 below.

File: rpm/kubelet.spec [hunk 2]
```diff
@@ -20,9 +20,14 @@ Source5: https://storage.googleapis.com/kubernetes-release/network-plugins/cni-a
 
 BuildRequires: curl
 Requires: iptables >= 1.4.21
+Requires: kubernetes-cni >= 0.3.0.1
```
Comment (dgoodwin) [inline rpm/kubelet.spec]: Is this a required package? I thought that was optional.

File: rpm/kubelet.spec [hunk 3]
```diff
@@ -20,9 +20,14 @@ Source5: https://storage.googleapis.com/kubernetes-release/network-plugins/cni-a
...
 Requires: socat
 Requires: util-linux
 Requires: ethtool
+Requires: iproute2
```
Comment (bjornl) [inline rpm/kubelet.spec]: Have this been tested? AFAIK there is not package named iproute2 on RHEL and CentOS 7. This dependency can not be meet.

Comment (dgoodwin) [inline rpm/kubelet.spec]: You are correct, found and fixed recently in https://github.com/kubernetes/release/pull/212 but this hasn't made it to unstable yet.

Comment (Starefossen) [inline rpm/kubelet.spec]: @dgoodwin any ETA on the unstable repo?

Comment (dgoodwin) [inline rpm/kubelet.spec]: Afraid I'm not sure, @mikedanese @luxas could we get a new unstable build going soon? It's currently quite busted on CentOS.

Comment (mikedanese) [inline rpm/kubelet.spec]: Sure, we can do one today

**Pull request merged**